crates/cargo-contract/src/cmd/build/mod.rs

```diff
@@ -668,20 +757,40 @@ pub(crate) fn execute(args: ExecuteArgs) -> Result<BuildResult> {
                 exec_cargo_dylint(&crate_metadata, verbosity)?;
             }
 
+            if let Some((x, y)) = counter {
+                maybe_println!(
+                    verbosity,
+                    "\n {} {}",
+                    "Checking contract".bright_purple().bold(),
+                    format!("[{}/{}]", x, y).bold(),
```

Comment:
Can we also add a counter to the output , something like this:

```
Results for contract `foo` [1/2]

Original wasm size: 48.5K, Optimized: 22.6K

The contract was built in DEBUG mode.

Your contract artifacts are ready. You can find them in:
/private/tmp/flipper_virtual/target/ink/foo

  - foo.contract (code + metadata)
  - foo.wasm (the contract's code)
  - metadata.json (the contract's metadata)

Results for contract `bar` [2/2]

Original wasm size: 48.5K, Optimized: 22.6K

The contract was built in DEBUG mode.

Your contract artifacts are ready. You can find them in:
/private/tmp/flipper_virtual/target/ink/bar

  - bar.contract (code + metadata)
  - bar.wasm (the contract's code)
  - metadata.json (the contract's metadata)
  ```

Reply:
there's no easy way to get the contract's name from `BuildResult` struct, so we're only printing the counter:
```
Results for contract [1/2]

Original wasm size: 63.0K, Optimized: 31.2K

The contract was built in DEBUG mode.

Your contract artifacts are ready. You can find them in:
/home/bear/develop/ink/examples/target/ink/erc20

  - erc20.contract (code + metadata)
  - erc20.wasm (the contract's code)
  - metadata.json (the contract's metadata)

Results for contract [2/2]

Original wasm size: 68.6K, Optimized: 34.4K

The contract was built in DEBUG mode.

Your contract artifacts are ready. You can find them in:
/home/bear/develop/ink/examples/target/ink/erc721

  - erc721.contract (code + metadata)
  - erc721.wasm (the contract's code)
  - metadata.json (the contract's metadata)
```